cardscan-base/src/main/java/com/getbouncer/cardscan/base/ScanActivityImpl.java

```diff
@@ -71,8 +72,15 @@ public void onClick(View v) {
         if (!mInDebugMode) {
             mDebugImageView.setVisibility(View.INVISIBLE);
         }
+
+        boolean showEnterCardManuallyButton = getIntent().getBooleanExtra(ENTER_CARD_MANUALLY_BUTTON, false);
+        TextView enterCardManuallyButton = findViewById(R.id.enterCardManuallyButton);
+        if (!showEnterCardManuallyButton) {
```

Comment:
consider reversing this logic. only make it visible if `showEnterCardManuallyButton` is `true`